smurfs/src/actions/index.js

```diff
@@ -13,3 +20,48 @@
    U - updateSmurf
    D - deleteSmurf
 */
+export const getSmurfs = () => {
+  return dispatch => {
+    dispatch({
+      type: FETCH_START
+    });
+    axios
+    .get("http://localhost:3333/smurfs")
+    .then(res => {
+      console.log("Res Data", res.data);
+      dispatch({
+        type: FETCH_SUCCESS,
+        smurfs: res.data
+      });
+    })
+    .catch(err => {
+      dispatch({
+        type: FETCH_ERROR, 
+        payload: err.response
+      });
+    });
+  }
+}
+
+export const addSmurf = (smurf) => {
+  return dispatch => {
+    dispatch({
+      type: ADD_SMURF
+    });
+    axios
+    .post("http://localhost:3333/smurfs", smurf)
+    .then(res => {
+      console.log("Res Data", res.data);
+      dispatch({
+        type: POST_SUCCESS,
+        smurfs: res.data
+      });
+    })
+    .catch(err => {
+      dispatch({
+        type: FETCH_ERROR, 
+        smurfs: err.response
+      });
+    });
+  }
+}
```

Comment:
Well done, your actions creator looks really clean and are easy to read, well use of different types as well.